agent/consul/server_test.go

```diff
@@ -164,8 +155,6 @@ func testServerConfig(t *testing.T) (string, *Config) {
 	config.ServerHealthInterval = 50 * time.Millisecond
 	config.AutopilotInterval = 100 * time.Millisecond
 
-	config.Build = "1.7.2"
```

Comment:
IIUC, there's one more instance of this in https://github.com/hashicorp/consul/blob/85fe875d0772a6da4b42c03795ab4b44e8f92182/agent/consul/leader_test.go#L1440 

these seem to be more like placeholders (gossiped around) than actual config knobs right?

https://github.com/hashicorp/consul/blob/eb21649f82bf78e27a75b012013f695d9beba6af/agent/consul/config.go#L176

Reply:
Oh ya, I think there are a few individual tests that do set this value. I think that's ok. We still want to be able to run some tests and pretend the agent is an older version. What I was fixing in this PR was defaulting to an old version. I think the default needs to be the current version.